pkgs/os-specific/linux/dcgm/default.nix

```diff
@@ -0,0 +1,108 @@
+{ lib
+, callPackage
+, gcc11Stdenv
+, fetchFromGitHub
+, addOpenGLRunpath
+, catch2
+, cmake
+, cudaPackages_10_2
+, cudaPackages_11_8
+, cudaPackages_12
+, fmt_9
+, git
+, jsoncpp
+, libevent
+, plog
+, python3
+, tclap_1_4
+, yaml-cpp
```

Comment:
nit: alphabetize

Reply:
I believe it is already alphabetized? I was under the impression that `lib`, `callPackage` and a few others were special cased to always be at the beginning.

Reply:
ah I see... we don't have an established style guide for this unfortunately. I often see `lib` and `stdenv` special cased, but the details are not worth bike shedding here